libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -778,58 +761,58 @@ bool AC_WPNav::set_spline_destination_loc(const Location& destination, const Loc
 bool AC_WPNav::set_spline_destination_next_loc(const Location& next_destination, const Location& next_next_destination, bool next_next_is_spline)
 {
     // convert next_destination location to NEU vector and determine if it uses terrain-relative altitude
-    Vector3p next_dest_neu_m;
+    Vector3p next_dest_ned_m;
     bool next_dest_is_terrain_alt;
-    if (!get_vector_NEU_m(next_destination, next_dest_neu_m, next_dest_is_terrain_alt)) {
+    if (!get_vector_NED_m(next_destination, next_dest_ned_m, next_dest_is_terrain_alt)) {
         return false;
     }
 
     // convert next_next_destination location to NEU vector and determine if it uses terrain-relative altitude
-    Vector3p next_next_dest_neu_m;
+    Vector3p next_next_dest_ned_m;
     bool next_next_dest_is_terr_alt;
-    if (!get_vector_NEU_m(next_next_destination, next_next_dest_neu_m, next_next_dest_is_terr_alt)) {
+    if (!get_vector_NED_m(next_next_destination, next_next_dest_ned_m, next_next_dest_is_terr_alt)) {
         return false;
     }
 
     // initialize next spline segment using converted waypoints
-    return set_spline_destination_next_NEU_m(next_dest_neu_m, next_dest_is_terrain_alt, next_next_dest_neu_m, next_next_dest_is_terr_alt, next_next_is_spline);
+    return set_spline_destination_next_NED_m(next_dest_ned_m, next_dest_is_terrain_alt, next_next_dest_ned_m, next_next_dest_is_terr_alt, next_next_is_spline);
```

Comment:
we're happy with the naming of, "next_dest_is_terrain_alt"?

Reply:
Yes I think so. For now.